server/app/views/components/Icons.java

```diff
@@ -525,7 +581,67 @@ public static SvgTag questionTypeSvg(QuestionType type) {
       default: // fall through
         icon = Icons.UNKNOWN;
     }
-    return svg(icon);
+    return new SvgTag()
+        .with(new UseTag().attr("href", "#svg-link-" + icon.name().toLowerCase(Locale.ROOT)));
+  }
```

Comment:
```suggestion
    return svg(icon).withId(makeSvgLinkId(icon));
  }
  
  /**
  * Generate the HTML ID for the icon
  */
  private String makeSvgLinkId(Icon icon) { 
    return String.format("svg-link-%s", icon.name().toLowerCase(Locale.ROOT));
  }
```

Sorry, I think I miscommunicated on the id. The one that is hidden, that is the actual SVG, needs an ID which can't have a `#` in the name. And it needs to be built the same with as the full SVG to maintain it's styling.

Below, where we create the link version we need the `#` before the id and it just has the `use` element.